sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -1051,7 +1051,8 @@ class StatisticsSuite extends StatisticsCollectionTestBase with TestHiveSingleto
 
   test("test statistics of LogicalRelation converted from Hive serde tables") {
     Seq("orc", "parquet").foreach { format =>
-      Seq(true, false).foreach { isConverted =>
+      // Botth parquet and orc will have Hivestatistics, both are convertable to Logical Relation.
+      Seq(true, true).foreach { isConverted =>
```

Comment:
This is to test when the conversion is on and off. We shouldn't change it.

Reply:
Right. i think some misunderstanding i will recheck into this. Thanks